elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -1,26 +1,29 @@
 defmodule Web.Policies.New do
   use Web, :live_view
-  alias Domain.{Resources, Actors, Policies}
+  import Web.Policies.Components
+  alias Domain.{Resources, Actors, Policies, Auth}
 
   def mount(params, _session, socket) do
     # TODO: unify this dropdown and the one we use for live table filters
     resources = Resources.all_resources!(socket.assigns.subject, preload: [:gateway_groups])
     # TODO: unify this dropdown and the one we use for live table filters
     actor_groups = Actors.all_groups!(socket.assigns.subject, preload: :provider)
+    providers = Auth.all_active_providers_for_account!(socket.assigns.account)
     form = to_form(Policies.new_policy(%{}, socket.assigns.subject))
 
     socket =
       assign(socket,
         resources: resources,
         actor_groups: actor_groups,
+        providers: providers,
         params: Map.take(params, ["site_id"]),
         resource_id: params["resource_id"],
         actor_group_id: params["actor_group_id"],
         page_title: "New Policy",
         form: form
       )
 
-    {:ok, socket, temporary_assigns: [form: %Phoenix.HTML.Form{}]}
```

Comment:
Just curious, was this `temporary_assigns` causing issues?

Reply:
@bmanifold it was with one of the approaches to dynamically add form elements, now I think it should be fine